### ISSUE TITLE
[a11y] 9.1.4.4 - Text scalable to 200 %

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5385]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
Note: This was tested with a browser window size of 1280 x 768 pixels and a zoom of 200%.

## Status: rather not fulfilled
- [ ] **Page 1 (Alle Dateien)**
When copying or moving elements, the left menu disappears and doesn’t show up anymore like the hamburger button on the other sides.

## Status: fulfilled
- [x] **Page 2 (Mit anderen geteilten Dateien)**
- [x] **Page 3 (Gelöschte Dateien)**
- [x] **Page 4 (Konto)**

**Comment by pascalwengerter** [comment]
@tbsbdr @kulmann this one needs a bit more radical changes I'm afraid. What was the reasoning of not having any nav (top nav & left sidebar) present when copying/moving resources?

**Comment by kulmann** [comment]
I don't know the reasoning. @LukasHirt maybe?

**Comment by kulmann** [comment]
IMO would be ok to include the topbar there as well.

**Comment by stevefrenzel** [comment]
@kulmann @pascalwengerter @tbsbdr

I'm not sure why a high zoom level is causing the `header` element with id `oc-topbar` to disappear. My guess is a function involving the window with, like this one:

```javascript
isSidebarVisible() {
      if (this.sidebarNavItems.length === 0) {
        return false
      }
      return this.windowWidth >= 1200 || this.appNavigationVisible
    }
```

I fiddled around with anything that could be related to it but didn't have any success. I think someone with more experience with this architecture should have a look and and find the parameter that's causing this weird behaviour?

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10727
